rise/Managers/NotificationManager.m

```diff
@@ -54,7 +54,7 @@ - (void) scheduleNotificationWithHour:(NSInteger *)hour withMinute:(NSInteger *)
     UNMutableNotificationContent *content = [UNMutableNotificationContent new];
     content.title = @"Time to stretch!";
     if ([PFUser currentUser] != nil) {
-        content.body = [NSString stringWithFormat:@"It's a new day %@, keep building stronger habits and complete your daily stretch.", [PFUser currentUser].username];
+        content.body = [NSString stringWithFormat:@"It's a new day %@, keep building stronger habits and complete your daily stretch.", [PFUser currentUser][@"displayName"]];
```

Comment:
Try to avoid hard-coded diction key. Declare static NSString *const kPFUserDisplayName = @"display_name"; either on top of this file, of in a separate helper class, then use here.